### DISCUSSION TITLE
BufferLineGoToBuffer

### DISCUSSION THREAD

**Discussion by StevenXL** [LazyVim/LazyVim#4335] [Q&A] [answered]
Hi. 

I'd like to map a key to use `BufferLineGoToBuffer`. I have this so far in my `plugins` directory:

```lua
return {
  {
    "akinsho/bufferline.nvim",
    opts = {
      options = {
        numbers = "ordinal",
      },
    },
    keys = {
      {
        "gb",
        function()
          require("bufferline").go_to(1)
        end,
        desc = "Goto Buffer",
      },
    },
  },
}

```

Unfortunately, I can't figure out how to parameterize the `1` shown above. This setup will always go to the first visible buffer.

**Comment by cerebrl** [comment, marked as answer]
I use this method for displaying the "ordinal" number of the open buffer: https://github.com/cerebrl/nvim/blob/main/lua/plugins/ui.lua#L18

I then use this to map to `Alt <buffer number>`: https://github.com/cerebrl/nvim/blob/main/lua/config/keymaps.lua#L6

This has worked well for me.
